wallet-new/src/Cardano/Wallet/API/Dev.hs

```diff
@@ -0,0 +1,9 @@
+module Cardano.Wallet.API.Dev where
+
+
+import           Servant ((:>))
+
+import           Cardano.Wallet.API.Types
+import qualified Cardano.Wallet.API.Dev.Test as Test
+
+type API = Tags '["Test"] :> Test.API
```

Comment:
We need to be consistent with names. If this is a development API, we shouldn't also throw the word "test" in, as that conveys a different endeavour.

Reply:
If you care about it, please make a suggestion how to name it.

Reply:
I suggest we simply have the entire API defined at the top-level here (so we move the content of `Dev.Test` into this module, and we get rid of the artificial subsplit. Finally, we change the `Tags` content to be just `Development`, for the sake of consistency. What do you think?